### PR TITLE
feat: Add batch connnect/disconnect operation

### PR DESCRIPTION
**Pull request by sharefe-demo**
1. Add batch connect/disconnect opertaion
2. Fix typescript typing problem

### CODE REVIEW

**Review by lihqi — CHANGES_REQUESTED** [review]

File: packages/lb-components/src/components/audioAnnotate/index.tsx [hunk 1]
```diff
@@ -576,7 +576,7 @@ const AudioAnnotate: React.FC<AppProps & IProps> = (props) => {
     <AudioClipProvider>
       <Spin spinning={loading} wrapperClassName='audio-tool-spinner'>
         <Layout className={getClassName('layout', 'container')} style={{ height: '100%' }}>
-          {props?.leftSider}
+          {typeof props.leftSider === 'function' ? props.leftSider() : props.leftSider}
```
[the remark this answers is not in the document]
Comment (sharefe-demo) [reply packages/lb-components/src/components/audioAnnotate/index.tsx]: 代码有错，这里其实为了修复typing

File: packages/lb-components/src/views/MainView/LLMLayout/index.tsx [hunk 2]
```diff
@@ -38,7 +38,7 @@ const LLMLayout: React.FC<AppProps & IProps> = (props) => {
           };
         }, [hoverKey, modelAPIResponse, newAnswerList])}
       >
-        {props?.leftSider}
+        {typeof props.leftSider === 'function' ? props.leftSider() : props.leftSider}
```
[the remark this answers is not in the document]
Comment (sharefe-demo) [reply packages/lb-components/src/views/MainView/LLMLayout/index.tsx]: 代码有错，这里其实为了修复typing

File: packages/lb-components/src/views/MainView/toolFooter/BatchSwitchConnectIn2DView/BatchSwitchConnectIn2DView.tsx [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+  const { cuboidBoxIn2DView } = useContext(PointCloudContext);
+  const { t } = useTranslation();
+
+  const confirm = useCallback((e: Parameters<Confirm>[0], isConnect: boolean) => {
+    // window.console.log('confirm: ', e, isConnect)
+    EventBus.emit(EventBusEvent.switchConnect, isConnect);
+  }, []);
+
+  const cancel = useCallback((e: Parameters<Confirm>[0], isConnect: boolean) => {
+    // window.console.log('cancel: ', e, isConnect);
```
[the remark this answers is not in the document]
Comment (sharefe-demo) [reply packages/lb-components/src/views/MainView/toolFooter/BatchSwitchConnectIn2DView/BatchSwitchConnectIn2DView.tsx]: done

File: packages/lb-components/src/views/MainView/toolFooter/BatchSwitchConnectIn2DView/BatchSwitchConnectIn2DView.tsx [hunk 4]
```diff
@@ -0,0 +1,88 @@
+import React, { FC, useCallback, useContext } from 'react';
```
[the remark this answers is not in the document]
Comment (sharefe-demo) [reply packages/lb-components/src/views/MainView/toolFooter/BatchSwitchConnectIn2DView/BatchSwitchConnectIn2DView.tsx:5]: done

**Review by lihqi — APPROVED** [review]

**Pull request merged**